### DISCUSSION TITLE
Wiki malware link

### DISCUSSION THREAD

**Discussion by mensab** [TriliumNext/Trilium#2933] [General]
The 'latest release' link on https://github.com/zadam/trilium/wiki/Desktop-installation
actually points to https://github.com/DlSTRHO/trilium/releases/tag/fac9fef
which looks like a clone of the real one.
But the trilium-windows-x64-0.52.3.zip extracts a 700mb trilium.exe, i'm guessing to make it too large for AV software to scan.

Looking at the page history that link has been changed several times to other URLs, maybe you should stop random people editing the wiki.

**Comment by zadam** [comment]
Hi, thanks for the report. I've fixed the links and blocked this user from further editing wiki.

Letting users to fill in informaton has been so far pretty useful, I hope this was an isolated incident.

**Reply by vgwidt** [reply]
Saw this, checked to see if it was tampered with again, and indeed it was.  The link now points to https://github.com/VundIe/trilium/releases/tag/fac9fef

I restored the link in the 3 pages.  Likely the same actor but maybe not likely to stop.

**Reply by sigaloid** [reply]
It's been edited again. Fixed. 

@zadam Can you consider locking the wiki page from editing?

**Reply by hewo** [reply]
fixed again

**Reply by zadam** [reply]
Thanks for help.

I restricted editing to collaborators. If you miss having write access, write to me.

**Reply by vgwidt** [reply]
Thanks @zadam.
The Privacy Policy page still has the bad link, and the Desktop Installation page should be set to https://github.com/zadam/trilium/releases/latest (points direct to 0.52.3 at the moment).
